Cargo.toml

```diff
@@ -9,6 +9,7 @@ exclude = ["assets/*", ".github"]
 
 [dependencies]
 byteorder = "*"
+chrono = "*"
```

Comment:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html   bare version requirements are dangerous. (I don't know who started with that :P)